src/compiler/program.ts

```diff
@@ -10,6 +10,7 @@ module ts {
     /** The version of the TypeScript compiler release */
     export let version = "1.5.0.0";
 
+    /** @param setParentNodes True to skip binding stage setting parent nodes (useful when only interested in syntactic parsing). */

```

Comment:
Put this into the format of 

```TypeScript
/**
 * @param ....
 */
```

Consider changing to

```
Whether or not parent references should be set on nodes. This is useful when
one desires parent references, but the semantic information of binding is unnecessary.
```

Also, @mhegazy and @CyrusNajmabadi, I thought that we wanted `setParentNodes` to be non-optional. Maybe it would be useful to make that change here.